api/webhook/validate_auth.go

```diff
@@ -52,7 +52,6 @@ var authLog = ctrl.Log.WithName("validate-auth")
 type AuthValidator struct {
 	enabled bool
 	client  client.Client
-	reader  client.Reader
```

Comment:
both `reader` and `client` are not used in `AuthValidator`, it seems we could remove them

Reply:
Fixed.